openvdb_points/unittest/TestPointDataLeaf.cc

```diff
@@ -845,6 +847,55 @@ TestPointDataLeaf::testIO()
 }
 
 
+void TestPointDataLeaf::testDropAttributes()
```

Comment:
This is quite a lot of code to test a function which is really just a wrapper. Although it means not achieving 100% code coverage, I suggest we rely on the TestAttributeSet::testAttributeSet() function to test appending and dropping attributes from the attribute set to minimise the extra code duplication.